### ISSUE TITLE
Pb to tflite not working

### ISSUE THREAD

**Issue by innodeltech** [eManPrague/tensorflow-python-example#1]
Script : 

tflite_convert   --graph_def_file=retrained_graph.pb   --output_file=retrained.tflite   --input_format=TENSORFLOW_GRAPHDEF  --output_format=TFLITE   --input_shape=1,224,224,3   --input_array=input   --output_array=final_result   --inference_type=FLOAT   --
input_data_type=FLOAT 

Error : 
Exception: We are continually in the process of adding support to TensorFlow Lite for more ops. It would be helpful if you could inform us.

Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: ADD, AVERAGE_POOL_2D, CONV_2D, DEPTHWISE_CONV_2D, FULLY_CONNECTED, SOFTMAX. Here is a list of operators for which you will need custom implementations: Dequantize.
